styx-api-service/src/main/java/com/spotify/styx/api/WorkflowResource.java

```diff
@@ -85,15 +83,8 @@ public WorkflowResource(Storage storage, DockerImageValidator dockerImageValidat
             rc -> patchState(arg("cid", rc), arg("wfid", rc), rc.request()))
     );
 
-    final List<Route<AsyncHandler<Response<ByteString>>>> sunsetRoutes = Collections.singletonList(
-        Route.with(
-            json(), "PATCH", BASE + "/<cid>/state",
-            rc -> patchState(arg("cid", rc), rc.request()))
```

Comment:
Should this `patchState()` method also be removed?

Reply:
Yeah, it should.

Reply:
Gone now.